demo/src/registerServiceWorker.js

```diff
@@ -0,0 +1,120 @@
+// In production, we register a service worker to serve assets from local cache.
```

Comment:
🛑 I assume this is third-party code? We should move this into a third-party directory of sorts and explain where it came from.

Reply:
I'm adding a warning message.  Unfortunately, the way create-react-app sets up a project, all files must be inside `src` (or some sub-directory).  I'd rather create sub-directories for different demo "pages" and keep the number of files in `src` proper relatively small.

Reply:
Ok! Maybe just inside `src/third-party`? That way it's clearer that it's not something that we created (and maybe use the README convention to explain where we got it from and how to update it).

Reply:
I'll add a README.md with how it go created, but I don't actually know how to update it.  I'm not sure create-react-app has a good update story. :\